src/socketio.quiver.js

```diff
@@ -1390,6 +1404,41 @@ QuiverSocialProvider.prototype.err = function(code) {
   return err;
 };
 
+/*
+ * Initialize this.logger_.
+ * @private
+ */
+QuiverSocialProvider.prototype.initLogger_ = function() {
+  this.logger_ = console;  // Initialize to console if it exists.
+  if (typeof freedom !== 'undefined' &&
+      typeof freedom.core === 'function') {
+    freedom.core().getLogger('[QuiverSocialProvider]').then(function(log) {
+      this.logger_ = log;
```

Comment:
this.logger_ is initialized to console at the beginning of initLogger_ in case of this race condition